setup.py

```diff
@@ -88,10 +85,6 @@
               "example\n"
         f.write(msg)
 
-DATA_DIR = os.path.dirname(data.__file__)
```

Comment:
Copying of "pool_transactions_sandbox" to "~/.plenum" is already done using "data_files" argument of "setup" function call.